### PR TITLE
Alert design doc updates

### PR DESCRIPTION
**Pull request by mcarrano**
Updates to text and images for Alerts and notifications design guidelines.

Closes: #1753 #1697

### CODE REVIEW

**Comment by mcarrano** [comment]
@red

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 1]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
 
 ## Elements
 ### Alert
-Alerts are used to communicate status information to the user without blocking other actions through the user interface. There are two styles that can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Regardless of the styling, the basic elements of all alerts are consistent. Since toast alerts should always be dismissable, only a subset of variations should be used.
+Alerts are used to communicate status information to the user without blocking other actions through the user interface. There are two styles that can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Regardless of the styling, the basic elements of all alerts are consistent.
 
-![alert types](./img/alert-types-2.png)
+<img src="./img/alert-types.png" alt="Alert types" width="868">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:14]: Alert types

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 2]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
...
-4. **Action (optional):** An action can be placed inside of an alert. Only a single action is supported. Note that the alert can include a close button or a single action, but not both.
-5. **Description (optional):** If a longer message is required, it can be included within the Description field.
+4. **Actions (optional):** One or more actions can be placed inside of an alert.
+5. **Description (optional):** If a longer message is required, it can be included within the Description field. This can also include a navigation link as an alternative to an action.
 
 #### Communicating status
-The status area and alert title are coded to communicate the severity of an alert.
+The status icon, background, and alert title are coded to communicate the severity of an alert.
+
+<img src="./img/alert-status.png" alt="alert styles" width="700">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:27]: Convert to self-closing img tag by adding `/` before the closing bracket:
`alert styles`

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 3]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
...
 
-In addition to static message content and links, interactive elements may optionally be included in the body of an inline alert when the user is being asked for some input or feedback.
+An inline alert should never block the user from interacting with other content on a page. For informational alerts, the user should be able to dismiss the alert by clicking the Close button on the alert. For error conditions, we recommend that the alert is dismissed only when the condition raising it has been cleared.
 
-![interactive inline alert](./img/inline-interactive-alert.png)
+In addition to static message content and links, interactive elements may optionally be included in the body of an inline alert to ask the user for further input or feedback.
 
-#### When to use
-Use an inline alert when you want to return feedback as the result of a user action within a page while allowing the user to continue working. Common uses for inline alerts include form validation or warning messages that should not interrupt the user’s flow.
+<img src="./img/inline-interactive-alert.png" alt="interactive inline alert" width="798">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:67]: Convert to self-closing img tag by adding `/` before the closing bracket:
`interactive inline alert`

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 4]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
...
+Toast alerts present special challenges for accessibility. Screen readers will announce the arrival and message content of an incoming toast alert, but user focus will not be placed there. This will make it difficult for these users to directly act on the alerts itself. Therefore, we recommend that any message containing actions be descriptive of how the can navigate to perform the action in the application. Messages that contain links should specify the location in text as well as including a simple text link. For example, a simple notification message might take the form:
 
-![non-accessible toast notification](./img/notification-not-accessible.png)
+<img src="./img/Notification-content-not-accessible.png" alt="Non-accessible alert example" width="615">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:100]: Convert to self-closing img tag by adding `/` before the closing bracket:
`Non-accessible alert example`

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 5]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
...
 
 A more accessible version of this message would look like this:
 
-![non-accessible toast notification](./img/notification-accessible.png)
+<img src="./img/Notification-content-accessible.png" alt="Non-accessible alert example" width="615">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:104]: Convert to self-closing img tag by adding / before the closing bracket:
`Non-accessible alert example`

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 6]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
...
 
 While this is more verbose, it has the advantage of being usable for both sighted and non-sighted users by providing instructions to screen reader users about where they can go to complete the action.
 
+## Examples
+
+#### Toast alerts
+Toast alerts always stack in the upper right of a page.
+
+<img src="./img/toast-alerts-desktop.png" alt="toast alerts in a page" width="990">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:113]: Convert to self-closing img tag by adding / before the closing bracket:
`toast alerts in a page`

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 7]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
...
+
+
+#### Inline alerts
+If an alert message applies globally to the content on a page, place the alert in the page header area just below the title.
+
+<img src="./img/Inline-alert-page-level.png" alt="global inline alert in a page" width="990">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:119]: Convert to self-closing img tag by adding / before the closing bracket:
`global inline alert in a page`

File: packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md [hunk 8]
```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
...
+
+But if the alert applies only to specific content, you might consider placing the alert inline with page content so the user can more easily find the problem.
+
+<img src="./img/Inline-alert-specific-context.png" alt="inline alert inline with content" width="990">
```
Comment (evwilkin) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md:123]: Convert to self-closing img tag by adding / before the closing bracket:
`inline alert inline with content`

**Comment by evwilkin** [comment]
> @redallen Looks like this PR is failing to build on V4. Any idea what the problem is?

@mcarrano the `` tags need to be closed, either with `</img>` after them or by making them self-closing by adding a slash before the closing bracket `` - I've added comments where these changes are needed.

FYI this is due to our docs using MDX (vs standard MD) which allows us to use React/JSX but which also expects each tag to be closed and throws these errors otherwise.

**Comment by redallen** [comment]
@evwilkin The build is failing because of the unclosed image tags.

**Pull request closed without merging**